openapi/components/schemas/UserSubscription.yaml

```diff
@@ -1,22 +1,5 @@
 type: object
 description: ''
-example:
-  id: vrchatplus-yearly
-  transactionId: txn_63231aa6-8b7a-49ff-a106-d5678f8318b1
-  store: Steam
-  steamItemId: '5000'
-  amount: 9999
-  description: VRChat Plus (Yearly)
-  period: year
-  tier: 5
-  active: true
-  status: active
-  expires: '2022-03-04T00:00:00.000Z'
-  created_at: '2020-12-04T13:09:54.344Z'
-  updated_at: '2020-12-10T19:53:45.362Z'
-  licenseGroups:
-    - lgrp_608513da-b213-4e15-80af-bd88c27f0979
-  isGift: false
```

Comment:
Object examples are easily out of date, I am in favour of removing them, but instead, let's put the example on the property itself. For example, the `id` below should have an example of `vrchatplus-yearly`.

Reply:
Can you tell me how to do examples on arrays and $ref types?

Reply:
You shouldn't need to put examples on the array itself or the reference, put it on the actual value.

Reply:
So I'd need to edit `TransactionID.yaml`, `SubscriptionPeriod.yaml`, `TransactionStatus.yaml` and `LicenseGroupID.yaml` and add an example there?

Reply:
For LicenseGroupID, yes, the rest are enums and are self-documenting.

Reply:
Okay, since `LicenseGroupID` already has an example, it seems like I'm good with the examples, if I just drop the last two commits.